CMakeLists.txt

```diff
@@ -189,6 +189,10 @@ include(${CMAKE_CURRENT_SOURCE_DIR}/cmake/kokkos_configure_trilinos.cmake)
 
 if(Kokkos_ENABLE_TESTS)
   find_package(GTest QUIET)
+  # If the low-memory tests are enabled, then set the relevant variable
+  if(Kokkos_ENABLE_LOW_MEM_TESTS)
+    set(KOKKOS_ENABLE_LOW_MEM_TESTS ON "Switch on Low-memory tests")
+  endif()
```

Comment:
Assuming that we would have an option, this is not right.
For one `KOKKOS_ENABLE_LOW_MEM_TESTS` is defined as a list (`"ON;Switch on Low-memory tests"`) which presumably wasn't the intent but in any case you would want to follow the way we register the `LARGE_MEM_TESTS` option, that is
https://github.com/kokkos/kokkos/blob/392b6d34ff2e688cd597115677266909906a587d/cmake/kokkos_enable_options.cmake#L83

Reply:
Also we would want to raise an error if both LARGE_MEM and LOW_MEM are enabled.